R/pkg/R/schema.R

```diff
@@ -69,10 +69,11 @@ structType.structField <- function(x, ...) {
 #' @param ... further arguments passed to or from other methods
 print.structType <- function(x, ...) {
   cat("StructType\n",
-      sapply(x$fields(), function(field) { paste("|-", "name = \"", field$name(),
-                                           "\", type = \"", field$dataType.toString(),
-                                           "\", nullable = ", field$nullable(), "\n",
-                                           sep = "") })
+      sapply(x$fields(), function(field) {
+             paste("|-", "name = \"", field$name(),
+                   "\", type = \"", field$dataType.toString(),
+                   "\", nullable = ", field$nullable(), "\n",
+                   sep = "") })
```

Comment:
It's not about style for comments but how about like as follows?
```
      sapply(x$fields(),
                 function(field) {
                   paste("|-", "name = \"", field$name(), 
                             "\", type = \"", field$dataType.toString(),
                             "\", nullable = ", field$nullable(), "\n",
                             sep = "")
                 })
```

@shivaram Do you have any suggestion?